backbone.js

```diff
@@ -1261,8 +1261,12 @@
 
     // Produces a DOM element to be assigned to your view. Exposed for
     // subclasses using an alternative DOM manipulation API.
-    _createElement: function(tagName) {
-      return document.createElement(tagName);
+    _createElement: function(tagName, namespace) {
+        if (typeof namespace === 'undefined') {
```

Comment:
sure, then even
```JavaScript
return namespace ?  document.createElementNS(namespace, tagName) : document.createElement(tagName);
```
Although if you are strict about comparisons with `undefined`, you would probably not want to do this.